### PR TITLE
Show merged PRs on myteam page

### LINKED ISSUE #55: Team PRs on myteam page
Currently the team PRs can only be seen if we open the team page through the leaderboard.
It will be easier for team members to see their PRs through my team dashboard.

I will implement this. :)

### PR DESCRIPTION
**Pull request by diwash007**
fixes #55

### CODE REVIEW

File: src/components/pages/team/hero/merged-prs.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                  )}
+                  {pr.title}
+                </p>
+                <p className="font-medium">
+                  <a href={pr.url} target="_blank" rel="noreferrer">
+                    <GitHubIcon className="h-[30px]" />
+                  </a>
+                </p>
+                {moderator && (
+                  <p className="cursor-pointer truncate font-medium" onClick={removePr(pr.id)}>
```
Comment (TheLearneer) [inline src/components/pages/team/hero/merged-prs.jsx:43]: The `removePr` function doesn't exists in the current component.
You might want to create that function definition in this file as well.

**Review by bsantosh909 — CHANGES_REQUESTED** [review]

File: src/components/pages/team/hero/merged-prs.jsx (hunk 1 above)
Comment (diwash007) [reply src/components/pages/team/hero/merged-prs.jsx:48]: Successfully fixed the issue.

File: src/components/pages/team/hero/team.jsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { toast } from 'react-toastify';
 import { useDebouncedCallback } from 'use-debounce';
 
 import GitHubIcon from '../../../../icons/github.inline.svg';
+import MergedPRs from './merged-prs';
```
Comment (TheLearneer) [inline src/components/pages/team/hero/team.jsx:10]: `Eslint error`: Import order

**Review by bsantosh909 — CHANGES_REQUESTED** [review]
Hi @diwash007 
Please make sure to run the project locally so that you can see the Eslint errors that are popping up.

Also, please make sure to update your code to the latest changes on upstream, because we have few changes done to the file you are changing and we don't want to miss out  🙏

File: src/components/pages/team/hero/team.jsx (hunk 2 above)
Comment (diwash007) [reply src/components/pages/team/hero/team.jsx]: What should be the correct order?
Parents before siblings .. isn't this correct?
image

File: src/components/pages/team/hero/team.jsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import { toast } from 'react-toastify';
 import { useDebouncedCallback } from 'use-debounce';
 
 import GitHubIcon from '../../../../icons/github.inline.svg';
+import MergedPrs from './merged-prs';
```
Comment (TheLearneer) [inline src/components/pages/team/hero/team.jsx:10]:
```suggestion

import MergedPrs from './merged-prs';
```

File: src/components/pages/teams/hero/hero.jsx [hunk 4]
```diff
@@ -7,13 +7,13 @@ import GitHubIcon from '../../../../icons/github.inline.svg';
 import DiscordIcon from './images/discord.inline.svg';
 import TwitterIcon from './images/twitter.inline.svg';
 
+import MergedPrs from 'components/pages/team/hero/merged-prs';
```
Comment (TheLearneer) [inline src/components/pages/teams/hero/hero.jsx:10]: Please remove this line from here and add before GithubIcon, make sure to add a empty line after this as well.

**Review by bsantosh909 — CHANGES_REQUESTED** [review]

**Comment by TheLearneer** [comment]
@nevo-david Do we want to show the option to remove the PR from `myteam` link or is it not necessary there?
Because if we need to do this, we will need to make little change on the backend that it will return the deleted status as well.

**Pull request closed without merging**